### ISSUE TITLE
Get the loss in LongformerForQuestionAnswering for fine-tuning

### ISSUE THREAD

**Issue by hanane-djeddal** [huggingface/transformers#12114]
Hello,

I'm trying to fine-tune **LongformerForQuestionAnswering** on a custom dataset. I've written a script for training (without using hugging face _Trainer_), and I need the loss of the model for that. On the Longformer docs page, it's written that : 

**loss** (torch.FloatTensor of shape (1,), optional, returned when labels is provided) – Total span extraction loss is the sum of a Cross-Entropy for the start and end positions.

Meaning that the model is supposed to return **loss** when the input _label_ is provided, however, the model takes no such an input (it's not mentioned in the doc, and the model generates an error when passing the input (label=...))

I've noticed that for **LongformerForMaskedLM** this is not an issue since the model does take _label_ as an input. 

I am wondering if there is a way to get the **loss** from LongformerForQuestionAnswering and perhaps to correct this on the docs page. 
Thanks!

**Comment by NielsRogge** [comment]
In HuggingFace Transformers, `xxxForQuestionAnswering` models don't take a `labels` argument as input. Rather, one should provide `start_positions` and `end_positions`. These indicate which token are the start of the answer, and which token are the end of the answer.

Check out this notebook which showcases how to fine-tune a model for question-answering: https://github.com/huggingface/notebooks/blob/master/examples/question_answering.ipynb
